src/Mimi.jl

```diff
@@ -10,7 +10,8 @@ export
     connectparameter, setleftoverparameters, getvariable, adder, MarginalModel, ConnectorCompVector,
     ConnectorCompMatrix, getindex, getdataframe, components, variables, getvpd, unitcheck, plot, getindexcount,
     getindexvalues, getindexlabels, delete!, get_unconnected_parameters, Timestep, isfirsttimestep,
-    isfinaltimestep, TimestepVector, TimestepMatrix, hasvalue, update_external_parameter
+    isfinaltimestep, TimestepVector, TimestepMatrix, hasvalue, update_external_parameter,
+    getdataframe_for_parameter, parameters, explore # added for explore.jl
```

Comment:
We probably shouldn't export ``getdataframe_for_parameter`` here, right? That seems more like an internal function. Same for ``parameters``, or do we want to make that part of the official API?

Also remove the comment here about ``explore.jl``.

Reply:
@davidanthoff  above we do export getdataframe and components, so I figured we'd do the same with the parameter versions, but definitely something we might want to discuss?

Reply:
getdataframe_for_parameter is going away. I've already modified getdataframe to handle both cases (in the dot-overloading branch). So better to not export it.